soccer/Robot.hpp

```diff
@@ -530,6 +532,9 @@ class OurRobot : public Robot {
     friend class MotionControl;
 
 private:
+    mutable RJ::Time _lastBallSense;
```

Comment:
@joshhting this is pretty hacky (per before) since we can't rely on having hasBall being called reliably all the time. Maybe we could move it into `setRadioRx` (https://github.com/RoboJackets/robocup-software/blob/master/soccer/Robot.hpp#L401-L404), which should be called on every incoming packet (I think). It's even in the same class, so it should be easy to move it over.